src/components/inputs/ValidatedTextInput.js

```diff
@@ -0,0 +1,83 @@
+import { debounce } from "lodash";
```

Comment:
You could try to sort imports according to the rules of importing in React. We could apply those rules in new components like this. But this is not obligatory to fix this in this PR. This is only a proposal.
In my opinion we should try to remember about this while adding a whole new files into the project. 
@olewandowski1

Reply:
I agree, it is definitely worth doing. I sorted imports according to the rules which you sent me on DMs.

Reply:
thank you for improving the order of imports!